### ISSUE TITLE
seems saturn run failed with ubuntu 20.04

### ISSUE THREAD

**Issue by guocf20** [filecoin-saturn/L1-node#132]
2022/11/29 06:57:31 [notice] 32#32: start cache loader process 60
  node-main:system Memory Total: 65.9 GB Free: 61.5 GB Available: 64.1 GB +0ms
  node-main:system Disk Total: 1353 GB Used: 6 GB Available: 1279 GB +10ms
  node-main:system CPUs: 16 (1.01, 0.52, 0.27) +1ms
  node-main:system NIC eth0: 235 GB downloaded / 137 GB uploaded +6ms
  node-main:registration All requirements met +0ms
  node-main:system Executing speedtest +0ms
  node-main:system Done executing speedtest. 7233.195288 Mbps DL / 9128.331983999999 Mbps  UL +10s
  node-main:registration Speed requirement met +10s
  node-main:registration Certificate is valid until Feb 27 23:59:59 2023 GMT +1ms
  node-main:registration Re-registering with orchestrator... +0ms
  node-main:registration:check Successful +0ms
  node-main:registration Node's geolocation is set to Singapore, Singapore, Singapore. If this is wrong, please open an issue at https://github.com/filecoin-saturn/L1-node/issues +3s
  node-main:registration Successful re-registration, updated token +0ms
  node-main:log-ingestor Reading nginx log file +0ms
2022/11/29 06:58:32 [notice] 60#60: http file cache: /usr/src/app/shared/nginx_cache 0.004M, bsize: 4096
2022/11/29 06:58:32 [notice] 32#32: signal 17 (SIGCHLD) received from 60
2022/11/29 06:58:32 [notice] 32#32: cache loader process 60 exited with code 0
2022/11/29 06:58:32 [notice] 32#32: signal 29 (SIGIO) received

**Comment by KlausGlueckert** [comment]
did you do a self-signed certificate?

**Comment by guocf20** [comment]
no, just follow the readme steps .  https://orchestrator.strn.pl/stats shows the node is Draining and weight became -1. and the data folder is empty.
